### DISCUSSION TITLE
Ability to edit tags of a message

### DISCUSSION THREAD

**Discussion by mynameismon** [edemaine/coauthor#592] [Q&A]
As it currently stands, tags can only be added to a message and not removed from it. However, sometimes it happens accidently that one selects a tag to be added. It would be easier if there could be an (x) over the tag to remove the tag as easily as one can add it

**Comment by edemaine** [comment]
That is already a feature: when editing a message, (x)s appear next to the tag, and clicking on it removes the tag from the message.
